### ISSUE TITLE
Windows 10 import error

### ISSUE THREAD

**Issue by MatejNikl** [kyamagu/faiss-wheels#42]
Hey, It seems like the same issue as #38 prevails even in 1.7.1:
```
    import faiss
C:\Python\lib\site-packages\faiss\__init__.py:18: in <module>
    from .loader import *
C:\Python\lib\site-packages\faiss\loader.py:65: in <module>
    from .swigfaiss import *
C:\Python\lib\site-packages\faiss\swigfaiss.py:13: in <module>
    from . import _swigfaiss
E   ImportError: DLL load failed while importing _swigfaiss: The specified module could not be found.
```
Windows 10, Python 3.9

**Comment by Holy-Shine** [comment]
same error.
python:     3.7.6
faiss-cpu:  1.7.1

**Comment by leonright** [comment]
After looking into `procmon`, I was able to resolve it by providing `openblass.dll` (had to rename it from `libopenblass.dll`). Shouldn't this `.dll` be included with the package?

**Comment by MatejNikl** [comment]
Thanks for the quick action and the effort you put into maintaining this package!

Unfortunately, the same issue is present even after your fix (i.e. in version `1.7.1.post1`).
Is there any additional information that I could provide that you would benefit from when fixing this issue?

**Comment by kyamagu** [comment]
@MatejNikl Alright, I don't have a windows host to debug this. Can you check what Dependency Walker shows?
https://support.pyxll.com/hc/en-gb/articles/360041392893-Error-importing-Python-module-DLL-load-failed

**Comment by kyamagu** [comment]
@MatejNikl Never mind. There was an issue in specifying repaired wheel in CI process. Will fix soon.

**Comment by kyamagu** [comment]
Fixed in #44

**Comment by MatejNikl** [comment]
Thanks for the quick fix :+1:
